src/main/java/online/book/store/repository/BookRepository.java

```diff
@@ -2,8 +2,6 @@
 
 import online.book.store.model.Book;
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.stereotype.Repository;
 
-@Repository
```

Comment:
Why did you remove repository annotation?

Reply:
It was conflict with another PR
Fixed